persistent/_timestamp.c

```diff
@@ -27,9 +27,40 @@ static char TimeStampModule_doc[] =
 "$Id$\n";
 
 
+
+#define SCONV ((double)60) / ((double)(0x10000)) / ((double)(0x10000))
```

Comment:
Ugh!  That name doesn't help with semantics at all.

Reply:
Renamed, and documented with concrete values.